tests/foreman/api/test_host.py

```diff
@@ -91,3 +93,244 @@ def test_positive_update_owner_type(self):
                 host.owner_type = owner_type
                 host = host.update(['owner_type'])
                 self.assertEqual(host.owner_type, owner_type)
+
+    @run_only_on('sat')
+    @tier1
+    def test_positive_create_with_name(self):
+        """Create a host with a random valid name
+
+        @feature: Hosts
+
+        @assert: A host is created and the name matches
+        """
+        for name in valid_hosts_list():
+            with self.subTest(name):
+                host = entities.Host()
+                host.create_missing()
+                host.name = name
+                host = host.create(create_missing=False)
```

Comment:
What is a specific reason to not have `host = entities.Host(name=name).create`?
Anyway that way of implementation doesn't seem similar to the code that we are usually use. Could you fix that?